research/src/pages/popup/popup.research.mdx

```diff
@@ -0,0 +1,11 @@
+---
+name: Popup
+path: /components/popup.research
+menu: Analysis
+---
+
+This research page will be expanded to detail various different subclasses of "popup" UI.
+
+## Additional research
+
+- [Initial proposal for a new `<popup>` element and related APIs](/components/popup.research.explainer)
```

Comment:
Can we just call this Explainer? Given that the term `proposal` (we should change this) in Open UI currently is used to denote the more normative spec text I'd prefer to not duplicate this name even though it is of course moving towards proposal.